src/tokens-dist/ts/colors.ts

```diff
@@ -146,7 +146,8 @@ export const EdsThemeColorTextGradeComplete = '#007249';
 export const EdsThemeColorTextGradeRevise = '#21272D';
 export const EdsThemeColorTextGradeStop = '#BD0044';
 export const EdsThemeColorTextDisabled = '#C0C4C8';
-export const EdsThemeColorTextBrandPrimary = '#5751D2';
+export const EdsThemeColorTextBrandDefault = '#5751D2';
+export const EdsThemeColorTextBrandPrimary = '#5751D2'; // deprecated
```

Comment:
nit questioning comment position for VSCode interaction

If you change this
```diff
- export const EdsThemeColorTextBrandPrimary = '#5751D2'; // deprecated
+ /** @deprecated will be removed in the next major upgrade? please use ... instead */
- export const EdsThemeColorTextBrandPrimary = '#5751D2';
```

Does the import statement look different and include the ~strikethrough~ text?

Reply:
Using the `@deprecated` tag does cause the token to have the ~~strikethrough~~ text but the comment is automatically added via `style-dictionary` from https://github.com/chanzuckerberg/edu-design-system/pull/1630/files#diff-845bd8692b0759a8fd359257750969e02a0c0591b4ab4eca7d6402fefc8e13bcR49 and will have to be re-added each time the tokens are built.

This `colors.ts` file is actually only used internally anyways for some stories and we should prob move away from this as well...